migration-scripts/tag_externalips_from_k8s.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <cluster-id>"
+  exit 1
+fi
+
+cluster_id=$1
+
+echo "Fetching ExternalIP resources from Kubernetes..."
+externalips=$(kubectl get externalips.kubestatic.quortex.io -o json)
+#echo $externalips
+
+# Count the number of ExternalIP resources in the cluster
+count=$(echo "$externalips" | jq '.items | length')
+echo "Found $count ExternalIP resources"
+
+# Loop over each ExternalIP resource and tag the corresponding EIP
+# with the instance ID of the node it is attached to
+# 
+# We encode the row in base64 to avoid issues with special characters
+# and decode it to extract the values we need
+for row in $(echo "${externalips}" | jq -r '.items[] | @base64'); do
+    # Helper function to extract values from the base64 encoded row
+    _jq() {
+        echo "${row}" | base64 --decode | jq -r "${1}"
+    }
+
+    name=$(_jq '.metadata.name')
+    allocation_id=$(_jq '.status.addressID')
+    node_name=$(_jq '.spec.nodeName')
+
+    # Skip ExternalIP resources that are not attached to a node
+    if [[ "$node_name" == "null" ]]; then
+        echo "Skipping $name no node_name, ExternalIP should be in reserved state"
+        continue
+    fi
+
+    echo "Resolving EC2 instance ID for node: $node_name..."
+    # Fetch the instance ID of the node the ExternalIP is attached to
+    instance_id=$(aws ec2 describe-instances \
+        --filters "Name=private-dns-name,Values=$node_name" \
+        --query "Reservations[].Instances[].InstanceId" \
+        --output text)
+
+    if [[ -z "$instance_id" ]]; then
+        echo "Could not find instance ID for node $node_name"
+        continue
+    fi
+
+    echo "    Tagging $allocation_id (EIP) with:"
```

Comment:
Maybe this log should not be indented to denote with the others ?

```suggestion
    echo "Tagging $allocation_id (EIP) with:"
```